08-transport.Rmd

```diff
@@ -143,10 +151,11 @@ region_ttwa = readRDS("extdata/bristol-ttwa.rds")
 
 The origin and destination zones used in this chapter are the same: officially defined zones of intermediate geographic resolution (their [official](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/bulletins/annualsmallareapopulationestimates/2014-10-23) name is Middle layer Super Output Areas or MSOAs).
 Each house around 8,000 people.
-Such administrative zones can provide vital context to transport analysis, such as the type of people who benefit particular interventions.
+Such administrative zones can provide vital context to transport analysis, such as the type of people who might benefit most from particular interventions <!-- reference --> .
 
-The geographic resolution of these zones is important: small zones with high geographic resolution are usually preferable but their high number in large regions can have consequences for processing (especially for origin-destination analysis in which the number of possibilities increases as a non-linear function of the number of zones).
-Another issue with small zones is related to anonymity rules: more socio-demographic variables are often available for large zones.
+The geographic resolution of these zones is important: small zones with high geographic resolution are usually preferable but their high number in large regions can have consequences for processing (especially for origin-destination analysis in which the number of possibilities increases as a non-linear function of the number of zones) <!-- reference -->.
+Another issue with small zones is related to anonymity rules, that means data will usually only be made available in such such a form that it is impossible to infer from them the identity of individuals. 
+In any case, fine-scale socio-demographic variables are often hard to collect or expensive which is why a common approach is to use sociodemographic data from coarser resolutions <!-- such as? And also think of the socio-demographic data we are using in the location chapter, 1 km^2 is already pretty fine and some data is even available for a 100 m resolution -->.
```

Comment:
Pls check if my rewrite captures the meaning you want to convey. Additionally, what are larger zones or coarser resolutions in this context, i.e., an example?